app/src/main/res/layout/item_autocomplete_emoji.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:gravity="center_vertical"
+    android:orientation="horizontal"
+    android:padding="8dp">
+
+    <ImageView
+        android:id="@+id/preview"
+        android:layout_width="40dp"
+        android:layout_height="40dp"
+        android:layout_marginStart="4dp"
+        android:layout_marginTop="4dp"
+        android:layout_marginEnd="4dp"
+        android:layout_marginBottom="4dp"
+        android:contentDescription="@null"
+        android:padding="4dp" />
+
+    <TextView
+        android:id="@+id/shortcode"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="8dp"
+        android:ellipsize="end"
+        android:maxLines="1"
+        android:textColor="?android:textColorPrimary"
+        android:textSize="?attr/status_text_large"
+        android:textStyle="normal|bold"
```

Comment:
Bold? It looks very different than the autocomplete of a hashtag, I think thats weird. But I actually like the bold, maybe add a dedicated autocomplete layout for hashtags that is bold as well?

Reply:
All right, I'll try it.

Reply:
I wasn't sure how it'd be, but I like the bold hashtag autocomplete a lot. Now all three kinds of autocomplete bold their results. :+1: